tbx/project_styleguide/templates/patterns/molecules/blog-item/blog-item.html

```diff
@@ -1,39 +1,25 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
-<a class="blog-item {% if classes %}{{ classes }}{% endif %}" href="{% firstof item.url item.link_external %}">
-    <h2 class="blog-item__title">{{ item.title }}</h2>
-    <span class="u-sr-only">,</span>
+<a href="{% firstof item.url item.link_external %}">
+    <h2>{{ item.title }}</h2>
 
     {% if item.intro %}
-        <p class="blog-item__intro">{{ item.intro }}</p>
-        <span class="u-sr-only">,</span>
+        <p>{{ item.intro }}</p>
     {% endif %}
 
     {% if item.description %}
-        <p class="blog-item__intro">{{ item.description|richtext }}</p>
-        <span class="u-sr-only">,</span>
+        <p>{{ item.description|richtext }}</p>
```

Comment:
Suggestion: We might not even use this but it needs to be wrapped in a `div` so not to output an empty `<p>` tag
```suggestion
        <div>{{ item.description|richtext }}</div>
```